docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -0,0 +1,95 @@
+.. _2017_cause_models:
+
+============
+Cause Models in GBD
+============
+
+What is a cause?
+----------------
+A cause in the GBD is something that is responsible for death, disease, or disability. 
+Some examples could be infectious diseases like malaria or diarrhea. 
+They could also be chronic diseases like diabetes or cancer. The could be things
+that people are born with like congenital disorders or happen during childbirth like maternal hemorrhage.
+Some causes are only associated with non-fatal outcomes, such as lower back pain.
+Many different things cause health loss and 
+this variation is reflected in the many ways that GBD models them. 
+
+Causes in the GBD are organized in a hierarchy where at every level of that hierarchy
+causes of death or disease are mutually exclusive and collectively exhaustive. 
+
+- All causes
+
+  - Injuries, communicable diseases, non-communicable diseases
+
+    - ... down to the most detailed level
+
+Some causes are only associated with non-fatal outcomes, such as lower back pain.
+
+Measures for causes include:  
+Deaths, incidence, prevalence, YLDs, YLLs, and DALYs
+	
```

Comment:
Should we also include metrics here? As in counts/rates/percent?

Reply:
I added that suggestion!